apps/settings/lib/Controller/AppSettingsController.php

```diff
@@ -581,7 +581,11 @@ public function uninstallApp(string $appId): JSONResponse {
 		$appId = $this->appManager->cleanAppId($appId);
 		$result = $this->installer->removeApp($appId);
 		if ($result !== false) {
-			$this->appManager->clearAppsCache();
+			/** @var \OC\App\AppManager $appManager */
+			$appManager = $this->appManager;
+			// If this app was force enabled, remove the force-enabled-state
+			$appManager->removeOverwriteNextcloudRequirement($appId);
+			$appManager->clearAppsCache();
```

Comment:
```suggestion
			// If this app was force enabled, remove the force-enabled-state
			$this->appManager->removeOverwriteNextcloudRequirement($appId);
			$this->appManager->clearAppsCache();
```
Is the property missing the typing?

Reply:
No but this is not a public method.

Reply:
Wdym?

Reply:
> $this->appManager->removeOverwriteNextcloudRequirement($appId)

Would end up in the baseline, so casting it fist to the private implementation.

Reply:
Ah ok, then ignore my comments.